docs/SPEC.md

```diff
@@ -2181,7 +2179,6 @@ from(bucket: "telegraf/autogen")
 #### Duplicate 
 
 Duplicate duplicates a specified column in a table.
```

Comment:
I guess this is consistent but it feels wrong.  the 'duplicate' of nothing is nothing...  and yet I feel like if I'm asking to duplicate something I should probably be told if that thing is not there.  

I would presume that it's OK to quietly move on here, since they'll likely hit the error on some subsequent code in the next step.  Unless they wrote back to storage...

Reply:
I removed the checks in duplicate and rename with the optimizer in mind. My think was that if one of the schema functions errors on missing columns then that reduces the space of optimizations we can make.

The example I'm thinking of is `schema_function -> filter`. If `schema_function` errors on missing columns then we can't push `filter` below it, because filter could potentially remove the table or tables that would have caused `schema_function` to error. The same holds for the reverse case. However the operations do commute if we remove the column checks.